pkg/cfn/manager/cluster.go

```diff
@@ -161,3 +157,14 @@ func getClusterName(s *Stack) string {
 	}
 	return ""
 }
+
+func getClusterNameTag(s *Stack) string {
+	for _, tag := range s.Tags {
+		if *tag.Key == api.ClusterNameTag {
+			if strings.HasSuffix(*s.StackName, "-cluster") {
```

Comment:
Should have removed this suffix check, that's all. Will open another PR, so that we keep this PR nicely scoped also.